File: 9-palindrome-number/9-palindrome-number.cpp
typedef long long int ll;
class Solution {
public:
    bool isPalindrome(int x) {
        int rev = 0;
        int num = x;
        while (num > 0) {
            int las = num % 10;
            rev = (ll)rev * 10 + las;
            num /= 10;
        }
        return x == rev;
    }
};

File: 9-palindrome-number/9-palindrome-number.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function(x) {
    let num = x
    rev = 0
    while (num > 0) {
        let las = num % 10
        rev = rev * 10 + las
        num = ~~(num/10)
    }
    return x == rev
};